### DISCUSSION TITLE
Error Message DCS Liberation

### DISCUSSION THREAD

**Discussion by Todd63** [Khopa/dcs_liberation#2154] [Q&A] [answered]
I know i messed up. I wanted to load the latest Liberation campaign but decided to delete the old one and did something wrong. The new one (5.2). Now when i press Fly i get an error message of (Permission denied. C:/Users/name/Saved Games/DCS/Liberation/Saves\Missions\\liberation_nextturn.miz')  I tried to change it to run as administrator but no luck. I tried removing it again and reinstalling with no luck.  Thanks.
![Screenshot 2022-04-07 085748](https://user-images.githubusercontent.com/103194427/162205361-4798d4b7-980a-435a-b5c2-b4507357298b.jpg)

![image](https://user-images.githubusercontent.com/103194427/162205448-dcb065b7-4ff1-494e-a1f4-80891277fdaf.png)

**Comment by oho111** [comment]
Did you find a solution. I too have the same problem after a new windows install.

**Reply by Starfire13** [reply, marked as answer]
The reason @Todd63 had this problem is because he set his paths wrong. That is the #1 reason why this occurs.

You can see that he had set his DCS Saved Games directory as:
C:\Users\td\Saved Games\DCS\Liberation\Saves\

It should instead be:
C:\Users\td\Saved Games\DCS\

If you are on open beta, then of course use the openbeta directory.

**Comment by Todd63** [comment]
Yes, I did get it working. But I don’t remember whit I did it’s been so long. I think I ran the dcs repair app and that might have did it. Sorry I am no help. Good luck.

**Comment by Todd63** [comment]
Gotcha, thanks.
